app/assets/stylesheets/components/_form.scss

```diff
@@ -0,0 +1,19 @@
+.field-label {
+  display: block;
+  margin-top: 1rem;
+}
+
+form button[type="submit"],
+form input[type="submit"] {
+  padding-left: 2rem;
+  padding-right: 2rem;
+  margin-bottom: 2rem;
+  width: 100%;
+}
+
+@media #{$breakpoint-sm} {
+  form button[type="submit"],
+  form input[type="submit"] {
+    width: auto;
+  }
+}
```

Comment:
The linter is flagging the following offenses:
```
6: Avoid qualifying attribute selectors with an element. [scss-lint]
6: Avoid qualifying attribute selectors with an element. [scss-lint]
8: Properties should be ordered margin-bottom, padding-left, padding-right, width [scss-lint]
15: Avoid qualifying attribute selectors with an element. [scss-lint]
15: Avoid qualifying attribute selectors with an element. [scss-lint]
```
If you don't agree with them, or need to disable some temporarily, please update the `.scss-lint.yml` file.